GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/CenterpieceViewModel.cs

```diff
@@ -0,0 +1,87 @@
+﻿using GalaxyZooTouchTable.Lib;
+using GalaxyZooTouchTable.Models;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Windows.Threading;
+
+namespace GalaxyZooTouchTable.ViewModels
+{
+    public class CenterpieceViewModel : ViewModelBase
+    {
+        public ObservableCollection<TableUser> AllUsers { get; set; } = new ObservableCollection<TableUser>();
+
+        private bool _showJoinMessage = true;
+        public bool ShowJoinMessage
+        {
+            get => _showJoinMessage;
+            set => SetProperty(ref _showJoinMessage, value);
+        }
+
+        private bool _flipCenterpiece;
+        public bool FlipCenterpiece
```

Comment:
I think this var should be called something like `CenterPieceIsFlipped`. `FlipCenterpiece` sounds more like a command.